influx-reports/influx-report.sh

```diff
@@ -1,6 +1,13 @@
+#!/bin/bash
 PYTHON_PATH="${WORKSPACE}/.venv/bin/python"
 REPORTS_FOLDER="${WORKSPACE}/test_results"
 INFLUX_DOWNTIME_FOLDER="$(dirname "${BASH_SOURCE[0]}")"
+ADDITIONAL_ARGS="$*"
 
+pushd "$WORKSPACE"
+  virtualenv .venv
```

Comment:
I added this because it didn't work without it but I can give it another go. So, just to be clear, you're saying I should be able find an existing venv in `$WORKSPACE/.venv`?